blocks/procedures.js

```diff
@@ -205,11 +234,7 @@ Blockly.Blocks['procedures_defnoreturn'] = {
       var varName = paramBlock.getFieldValue('NAME');
       this.arguments_.push(varName);
       var variable = this.workspace.getVariable(varName, '');
-      if (variable != null) {
```

Comment:
This was added by #2186 to fix #1958, but fields are happy now so this is unnecessary. Tests cover this change.